pom.xml

```diff
@@ -69,8 +69,8 @@
         <version.jcommander>1.72</version.jcommander>
         <version.jetty>6.1.26</version.jetty>
         <version.jgroups>4.0.19.Final</version.jgroups>
-        <version.jjwt>0.9.1</version.jjwt>
-        <version.junit>4.13.1</version.junit>
+        <version.jjwt>0.11.2</version.jjwt>
```

Comment:
as it turns out the authorization service upped the jwt version requiring me to update them here to allow using it in remote query service.